xbmc/guilib/GUIFontCache.cpp

```diff
@@ -32,6 +32,10 @@ class CGUIFontCacheImpl
     using HashIter = typename HashMap::iterator;
     using AgeMap = std::multimap<size_t, HashIter>;
 
+    ~EntryList()
+    {
+      Flush();
+    }
```

Comment:
According to the AddressSanitizer/LeakSanitizer there was non deleted memory after closing Kodi. So a code path not calling Flush() is very likely.